Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -21814,6 +21870,14 @@
 			],
 			"levels": 1,
 			"points_per_level": -5,
+			"features": [
+				{
+					"type": "reaction_bonus",
+					"situation": "from others aware of your reputation",
```

Comment:
It'd be nice if this could be more specific but this is probably the best that's currently practical. I've made a ticket for one possible way to improve that https://github.com/richardwilkes/gcs/issues/660